### PR TITLE
Fix: spacing between cards

### LINKED ISSUE #3759: [UI] spacing between cards should be corrected
### Current Behavior
spacing between cards is not correct

cards-error

### Desired Behavior
desired

### Implementation
Add margin-bottom to the cards

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by swagataroy30**
**Description**

This PR fixes #3759 

**Notes for Reviewers**

## Before 
Screenshot from 2023-02-20 01-19-05

## After
Screenshot from 2023-02-20 01-16-55

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 37462591b17bc12c08c8173e140031466c8f7cda at: https://63f2866e464f7c2c77e9f866--layer5.netlify.app

**Comment by leecalcote** [comment]
Tagging @avikt18 and @avi-88 for review.

**Review by Avi-88 — CHANGES_REQUESTED** [review]
It looks good. This does solve the issue at hand but  , I was just curious about the implementation of the cards as it would've been much better if we used css grid instead of just using flexbox with random margins and paddings to make it fit. Let me know what you think about that. Also while you are at it could you please change the border radius of the card to be a value instead of percentage. It's looking wierd for a single column layout for certain screen sizes like so

image

I guess 10px should roughly provide the same effect instead of 4% , thanks

**Comment by swagataroy30** [comment]
> It looks good. This does solve the issue at hand but , I was just curious about the implementation of the cards as it would've been much better if we used css grid instead of just using flexbox with random margins and paddings to make it fit. Let me know what you think about that. Also while you are at it could you please change the border radius of the card to be a value instead of percentage. It's looking wierd for a single column layout for certain screen sizes like so
> 
> image
> 
> I guess 10px should roughly provide the same effect instead of 4% , thanks

Hey sorry for the late reply ... Thanks @Avi-88 for the suggestions will change the border radius as said.

**Comment by Chadha93** [comment]
@swagataroy30  Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

**Review by avikt18 — COMMENTED** [review]
@swagataroy30 I guess making changes to a reusable component (`src/reusecore/Layout/Col.js` in this case) is not the best thing you can do. You can make changes to the projectSection.style.js instead. 
@Chadha93 pls correct me if I am wrong :sweat_smile:

**Comment by swagataroy30** [comment]
Removed changes that was done to the reusable component and moved changes to projectSection.style.js ... thanks @avikt18 for the suggestion. Fixed space between cards and also the border radius.

**Comment by l5io** [comment]
🚀 Preview for commit 707276a1ca3eed67ce01e58ea8c1c1ddca3e93c5 at: https://63fd030ca32b760a15fb9fc3--layer5.netlify.app

**Comment by swagataroy30** [comment]
Screenshot from 2023-02-28 00-41-16

Hey as you can see the below media query is a duplicate for 992px and also the css mentioned in it seems to be unused ... should i remove it ?

**Comment by Nikhil-Ladha** [comment]
> Screenshot from 2023-02-28 00-41-16
> 
> Hey as you can see the below media query is a duplicate for 992px and also the css mentioned in it seems to be unused ... should i remove it ?

Please merge the duplicate media queries into one, and if you think the CSS is unused feel free to remove it and push the changes.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Pending incorporation of Nikhil's feedback.

**Comment by swagataroy30** [comment]
> Pending incorporation of Nikhil's feedback.

Sorry for the late reply was busy with my exams. Made the said changes!

**Comment by l5io** [comment]
🚀 Preview for commit 1b3ac07921c83ba719b9a6c94e3dd66664c2fa73 at: https://6415fb0bdf97a603c92dfa49--layer5.netlify.app

**Review by Udit-takkar — COMMENTED** [review]
@swagataroy30 @Nikhil-Ladha has this been already fixed in some other PR?

This is what i see on https://layer5.io/

Screenshot 2023-04-10 at 2 59 47 AM

**Comment by swagataroy30** [comment]
But the issue still persists as i see..
Screenshot from 2023-04-10 08-07-37

File: src/sections/Home/Projects-home/projectSection.style.js [hunk 1]
```diff
@@ -111,11 +106,8 @@ const ProjectItemWrapper = styled.section`
         .project__block__wrap:before {
             content: none;
         }
-    }
-    @media only screen and (max-width: 992px) {
-        .project-text{
-            font-size: 7rem;
-            color:red;
+        .project__block__inner {
+            margin-bottom: 40px;
```
Comment (Nikhil-Ladha) [inline src/sections/Home/Projects-home/projectSection.style.js:110]: Make it `2.5rem` instead

Comment (swagataroy30) [reply src/sections/Home/Projects-home/projectSection.style.js]: Done!

**Comment by l5io** [comment]
🚀 Preview for commit ffd36fd9e1879785edbfd523044c9025589dadcf at: https://6434cec4db97f7611014848f--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**